frontend/package.json

```diff
@@ -27,7 +27,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "fmt": "prettier --check ."
```

Comment:
NIT: I'd love this to be called `fmt-check`, to be clear it won't format

Reply:
Yes, I think that's ideal too